glean-core/preview/examples/prototype.rs

```diff
@@ -0,0 +1,40 @@
+use std::env;
```

Comment:
Please add a license header here.

Reply:
Woops, that one got lost in a rebase, adding it back.